pubspec.yaml

```diff
@@ -15,10 +15,11 @@ dependencies:
   collection: ^1.1.0
   matcher: ^0.12.3
   meta: ^1.0.4
-  test: ^1.2.0
+  test_api: ^0.2.0
```

Comment:
0.2.1, if it won't get pulled in by the caret

Reply:
Done.